fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -2949,6 +2972,14 @@ public void dropMaterializedView(DropMaterializedViewStmt stmt) throws DdlExcept
                     baseTable.removeRelatedMaterializedView(table.getId());
                 }
             }
+            MaterializedView materializedView = (MaterializedView) table;
+            if (materializedView.getRefreshScheme().getType() == MaterializedView.RefreshType.ASYNC) {
```

Comment:
what if the refresh type is MANUAL? it seems that it should also drop the task if it exists.

Reply:
Drop task need taskId，currently I only know how to get the async task id.